lib/tasks/import/log.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+desc 'Import daily log from Access CSV at data/daily_log.csv'
+namespace :import do
+  task log: :environment do
+    CSV.foreach('data/daily_log.csv', encoding: 'ISO-8859-1', headers: true).with_index 2 do |row, line|
+      date = Date.parse row.fetch('Date:')
+      user_name = row.fetch('Dispatcher:')
+      if user_name.is_a? String
+        last_name, first_name = user_name.split(', ')
+        user = User.find_by name: [first_name, last_name].join(' ')
```

Comment:
Correct. More infuriatingly, existing dispatchers are referenced by name in the table — so if non-existing dispatchers were referenced by name too, we could do that. But they're just IDs, hence my check for `is_a? String`. So it's either a name, from which I have to do a DB lookup, or an ID that doesn't help me. 🎉